content/tutorials/publish-subscribe.textile

```diff
@@ -144,14 +166,30 @@ blang[ruby].
   "See this step in Github":https://github.com/ably/tutorials/commit/75becbf
 
 blang[swift].
+  If you never had a chance to build a basic user interface in Xcode, please refer to "Apple developer guide":https://developer.apple.com/library/content/referencelibrary/GettingStarted/DevelopiOSAppsSwift/Lesson2.html on how to build a simple UI and also learn more about "adding @@IBOutlets@ and @@IBActions@":https://developer.apple.com/library/content/referencelibrary/GettingStarted/DevelopiOSAppsSwift/Lesson3.html#//apple_ref/doc/uid/TP40015214-CH22-SW1.
+  To be able to subscribe to a channel add a button that will start subscription to "sports" channel. Go to your @ExampleViewController@ and add an @UIButton@ from @Object library@. Name the action "subscribeAction".
+  <a href="/images/tutorials/tutorials-swift-binding-UI-publish-subscribe.gif" target="_blank">
+    <img src="/images/tutorials/tutorials-swift-binding-UI-publish-subscribe.gif" style="width: 100%" alt="Binding UI objects">
+  </a>
+  In similar fashion add two @UILabels@ and connect one of them so that it will show the last message received. Name it "receivedMessageText".
+  Your @ExampleViewController@ should look similar to that view:
+  <a href="/images/tutorials/tutorials-swift-publish-subscribe-step3.png" target="_blank">
+    <img src="/images/tutorials/tutorials-swift-publish-subscribe-step3.png" style="width: 60%" alt="Binding UI objects">
+  </a>
+  Add this code to previously added @@IBAction@:
+
   ```[swift]
-    let channel = client.channels.get("sport")
+    let channel = ARTRealtimeChannel()
+    channel = client.channels.get("sport")
+    self.receivedMessageText.text = ""
+
     channel.subscribe { message in
-      print("Received \(message.data)")
+        self.receivedMessageText.text = "\(message.data)"
```

Comment:
Perhaps add a timestamp so that if the same message is published again it will be obvious something has changed?